spec/lib/saml_idp/controller_spec.rb

```diff
@@ -30,18 +30,28 @@ def params
       saml_response = encode_response(principal)
       response = OneLogin::RubySaml::Response.new(saml_response)
       response.name_id.should == "foo@example.com"
-      response.issuer.should == "http://example.com"
+      response.issuers.first.should == "http://example.com"
```

Comment:
Ah this is from the external OneLogin lib. That's out of our control then. As long as that's not an API change on our part, we're good.